docs/docs/themes/getting-started.md

```diff
@@ -1,35 +1,56 @@
 ---
-title: Getting Started
+title: Using a Gatsby theme
 ---
 
-Scaffolding out a project that uses themes requires you to use a starter that's implemented with theming.
+To get started using themes, we'll use a starter that installs the official gatsby blog theme.
```

Comment:
```suggestion
To get started using themes, you can use a starter that installs the official Gatsby blog theme.
```
Rephrasing slightly to follow the [Gatsby Style Guide](https://www.gatsbyjs.org/contributing/gatsby-style-guide/#use-you-as-the-pronoun).